config-linux.md

```diff
@@ -699,25 +704,19 @@ The following parameters can be specified to set up seccomp:
 }
 ```
 
-### <a name="seccompstate" />The Seccomp State
+### <a name="containerprocessstate" />The Container Process State
 
-The seccomp state is a data structure passed via a UNIX socket.
-The container runtime MUST send the seccomp state over the UNIX socket as regular payload serialized in JSON.
-The container runtime MUST also send the file descriptor(s) via `SCM_RIGHTS`: the seccomp file descriptor returned by the seccomp syscall and, optionally, the process file descriptor (e.g as returned by `pidfd_open(2)` or by `clone(2)` with the `CLONE_PID` flag).
+The container processs state is a data structure passed via a UNIX socket.
+The container runtime MUST send the container processs state over the UNIX socket as regular payload serialized in JSON and file descriptors MUST be send using `SCM_RIGHTS`.
 The container runtime MAY use several `sendmsg(2)` calls to send the aforementioned data.
 If more than one `sendmsg(2)` is used, the file descriptors MUST be sent only in the first call.
 
-The seccomp state includes the following properties:
-
-* **`ociVersion`** (string, REQUIRED) is version of the Open Container Initiative Runtime Specification with which the seccomp state complies.
-* **`fds`** (map, REQUIRED)
-    * **`seccompFd`** (int, REQUIRED) is the index of the file descriptor in the `SCM_RIGHTS` array refering to the seccomp notify file descriptor.
-    The value MUST be 0.
-    * **`pidFd`** (int, OPTIONAL) is the index of the file descriptor in the `SCM_RIGHTS` array referring to the target process file descriptor.
-    If present, this value MUST NOT be zero.
-* **`pid`** (int, REQUIRED) is the process ID, as seen by the runtime, on which the seccomp filter is applied (target process).
-* **`metadata`** (string, OPTIONAL) is the string set in `listenerMetadata`.
-    If the `listenerMetadata` is set, then the runtime MUST set this field too.
+The container processs state includes the following properties:
+
+* **`ociVersion`** (string, REQUIRED) is version of the Open Container Initiative Runtime Specification with which the container processs state complies.
+* **`fdIndexes`** (map, OPTIONAL) are the indexes of the file descriptors in the `SCM_RIGHTS` array.
```

Comment:
The change from `fds` to `fdIndexes` is not done in the "Example" below, line 727.

Reply:
Thanks for catching this up!